docs/en/04_Changelogs/4.1.0.md

```diff
@@ -0,0 +1,87 @@
+# 4.1.0
+
+## Introduction
+
+This version introduces many breaking changes, which in most projects can be managed through a combination
+of automatic upgrade processes as well as manual code review. This document reviews these changes and will
+guide developers in preparing existing 3.x code for compatibility with 4.0
+
+## Overview {#overview}
+
+ * Support for public webroot folder `public/`
+ * Better support for cross-platform filesystem path manipulation
+
+## Upgrading
+
+### Upgrade `public/` folder
+
+By default all new projects installed from 4.1.0 will have a `public/` webroot folder. This folder name is
+not configurable, but is turned on by creating this file, and off by ensuring this folder doesn't exist.
+
+When separating the public webroot from the BASE_PATH it is necessary to move a few files during migration:
+
+ - Move `.htaccess` from base to `public/`
+ - Move `index.php` from base to `public/`
+ - Move `assets` folder (including the nested `assets/.protected` folder) into `public/`
+ - Ensure that the `public/resources` folder exists; If this folder already exists in root, you should
+   delete this, and re-generate it by running `composer vendor-expose` in your root path.
+ - Any public assets committed directly to your project intended to be served directly to the
+ webserver. E.g. move `mysite/javascript/script.js` to `public/javascript/script.js`.
+ - Ensure that the web-root configured for your server of choice points to the public/ folder instead of the base path.
+ 
+You may also need to add various changes to your code if you reference the BASE_PATH directly:
+
+ - You should use `Director::publicFolder()` instead of `Director::baseFolder()` if referring to the
+ public folder.
+ - You can check if a public folder exists with `Director::publicDir()`
+ 
+### Use new `$public` theme set
```

Comment:
* Should contain a section on how to run your project without using themes, as an alternative or addition to "use new $public theme set"

Reply:
We probably need to discuss this; Most sites will have at least one theme.